Automattic/wp-calypso
# Collect RUM data and send it to logstash

#### Changes proposed in this Pull Request

* Adds a library to gather RUM data for performance and send it to Logstash
* Builds on top of #41539: the difference is that instead of Lux we use our own lib now.

#### Testing instructions

* Checkout the branch and start it with `yarn start`
* Navigate to `Plans`. You should see a request to `/logstash` in the network inspector. Verify the payload contains data.
* Go to logstash and query for `message:"perf.nav"` and check the data is there.
* Feature flag protection: Add `?flags=-rum-tracking/logstash` to the URL, verify there is no request to logstash
